vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -63,62 +67,90 @@ def init_job_with_gcloud(
+        "test_v3_1k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-1k-output.ht",
+        "test_v3_10k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-10k-output.ht",
+        "test_v3_100k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-100k-output.ht",
     }
 
-    # Reads in Hail Table, partitions, and exports to sharded VCF within the folder to be mounted
+    check_resource_existence(
+        output_step_resources={
+            "vrs-annotation-batch-wrapper.py": [
+                f"gs://{working_bucket}/vrs-temp/outputs/VRS-{prefix}",
+                output_paths_dict[args.version],
+            ]
+        },
+        overwrite=args.overwrite,
+    )
+
+    # Read in Hail Table, partition, and export to sharded VCF within the folder to be mounted
     ht_original = hl.read_table(input_paths_dict[args.version])
 
-    # Option to downsample for testing, if you want to test on v3.1.2 but not all of it
-    if args.downsample < 1.00 and args.version == "3.1.2":
+    # Option to downsample for testing, if you want to annotate part of a Hail Table but not all of it
+    if args.downsample and args.downsample < 1.00:
+        logger.info("Downsampling Table...")
         ht_original = ht_original.sample(args.downsample)
 
-    # Select() removed all non-key rows - VRS-Allele here is then added back to original table
-    ht = (
-        ht_original.select()
-    )  # QUESTION: is this okay or would this be cluttering the namespace ?
+    # Use 'select' to remove all non-key rows - VRS-Allele here is then added back to original table
+    ht = ht_original.select()
 
+    logger.info("Table read in and selected")
+
+    """
+    # Delete this block comment before merging - I do want to leave it in for now (as of 03/29) to keep track of progress at least
     # NOTE: REPARTITION DOES NOT OUTPUT EVENLY? LAST SHARD IS 5-TIMES THE SIZE OF OTHERS (STORAGE) AND MANY TIMES THE NUMBER OF VARIANTS
     # REACHED OUT TO HAIL TEAM ABOUT ISSUE, IN PROGRESS
     # 03-21 Response from Tim Poterba: an issue since the TESTING TABLES were sampled and subsetted randomly
     # --> Release Data is not affected in the same way
     # UPDATE: Tim has diagnosed problem, not yet fixed it yet on Query-On-Batch
-    ht = ht.repartition(args.partitions_for_vcf_export)
+    # UPDATE 03-28: All I had to do is Repartition them in a separate Notebook running Spark and re-output them
+    # It's fine AND everything matches the trush set from the VRS team!!! woo!!!
+    """
+
+    # Repartition the Hail Table if requested. In the following step, the Hail Table is exported to a Sharded VCF with one Shard per Partition
+    if args.partitions_for_vcf_export:
+        ht = ht.repartition(args.partitions_for_vcf_export)
 
     hl.export_vcf(
         ht,
-        f"gs://{working_bucket}/vrs-temp/shard-{prefix}.vcf.bgz",
+        f"gs://{working_bucket}/vrs-temp/shard-{args.version}.vcf.bgz",
```

Comment:
add 'version=args.versions' below 'working_bucket = args.working_bucket' and change 'args.version' to 'version' throughout the script

Reply:
will include in the next round of commits!